.github/workflows/python_test.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Install dependencies
         run: |
           poetry install --with dev,lint
-          poetry run pip install -U langchain
+          poetry run pip install -U langchain langchain-core
```

Comment:
Are there unit tests that are internal only without any extras from langchain/langchian-core?

Reply:
Most of the unit tests are that way - only a handful use langchain code